pkg/text/dictionary.go

```diff
@@ -0,0 +1,26 @@
+package text
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/fastly/cli/pkg/common"
+	"github.com/fastly/go-fastly/fastly"
+	"github.com/segmentio/textio"
+)
+
+// PrintDictionary pretty prints a fastly.Dictionary structure in verbose
+// format to a given io.Writer. Consumers can provider an prefix string which
```

Comment:
```suggestion
// format to a given io.Writer. Consumers can provider a prefix string which
```

Reply:
Just realized `provider` should be `provide` which makes me think this exists widely throughout the code because I likely would have copied this from elsewhere.